cpp/src/arrow/dataset/file_rados_parquet.h

```diff
@@ -197,18 +202,39 @@ class ARROW_DS_EXPORT RadosParquetFileFormat : public ParquetFileFormat {
   std::shared_ptr<DirectObjectAccess> doa_;
 };
 
+/// \brief Serialize the scan request.
+/// \param[in] options Scan options.
+/// \param[in] file_size File size.
+/// \param[out] bl Output buffer.
+/// \return Status.
 ARROW_DS_EXPORT Status SerializeScanRequest(std::shared_ptr<ScanOptions>& options,
                                             int64_t& file_size, ceph::bufferlist& bl);
 
+/// \brief Deserialize the scan request.
+/// \param[out] filter Filters.
+/// \param[out] partition Partition expression.
+/// \param[out] projected_schema The projection schema.
+/// \param[out] dataset_schema The dataset schema.
+/// \param[out] file_size File size.
+/// \param[in] bl Input Ceph buffer.
+/// \return Status.
 ARROW_DS_EXPORT Status DeserializeScanRequest(compute::Expression* filter,
                                               compute::Expression* partition,
                                               std::shared_ptr<Schema>* projected_schema,
                                               std::shared_ptr<Schema>* dataset_schema,
                                               int64_t& file_size, ceph::bufferlist& bl);
 
+/// \brief Serialize the table.
```

Comment:
Serialize the result Table to a bufferlist

Reply:
Done.